multi-int-store/src/main/java/com/connexta/multiintstore/callbacks/CallbackValidator.java

```diff
@@ -37,7 +37,7 @@ public Object parse(JsonNode body) {
 
     LOGGER.warn("{} is not a valid callback", body);
 
-    return null;
+    throw new UnsupportedCallbackException(body.toString());
```

Comment:
✏️ Should the exception contain a message like "Callback is not supported" instead of the body of the json?

Reply:
I mean it could, but it also has that in the name. I'm indifferent tbh